torch/csrc/jit/passes/frozen_ops_to_mkldnn.cpp

```diff
@@ -917,6 +961,16 @@ class MKLDNNSubgraphSlicer {
         return false;
       }
     }
+
+    if (n->matches(
+            "aten::layer_norm(Tensor input, int[] normalized_shape, Tensor? weight=None, Tensor? bias=None, float eps=1e-05, bool cudnn_enable=True) -> Tensor") &&
+        n->namedInput("weight")->type() != NoneType::get() &&
```

Comment:
technically we should be checking that the type is tensor since it could be OptionalType (unlikely since the parameters are likely frozen, but still)

Reply:
I think we do this in a few other places as well. Let me land the layernorm first and I'll fix all those places wholesale.